### PR TITLE
Added MD5 id and made forcemerging optional as it is time consuming.

### PR DESCRIPTION
**Pull request by cdahlqvist**

### CODE REVIEW

**Comment by cdahlqvist** [comment]
@dliappis Added a separate warmup phase and adjusted the number of clients to make sure the number of iterations can be evenly divided across them.

File: README.md [hunk 1]
```diff
@@ -208,13 +212,15 @@ The following document id types are benchmarked:
...
+Note that this challenge will generate up to ~400GB of data on disk and will require additional space for merging and overhead. Make sure around 500GB of disk space is available before running this to be on the safe side.
 
 The table below shows the track parameters that can be adjusted along with default values:
 
 | Parameter | Explanation | Type | Default Value |
 | --------- | ----------- | ---- | ------------- |
-| `bulk_indexing_clients` | Number of bulk indexing clients/connections | `int` | `20` |
+| `bulk_indexing_clients` | Number of bulk indexing clients/connections | `int` | `25` |
+| `bulk_indexing_iterations` | Number of bulk requests to send as part of each run | `int` | `200000` |
+| `forcemerge` | Parameter indicating whether index statistics should be gathered following aforcemerge down to a single segment | `string` | `no` |
```
[the remark this answers is not in the document]
Comment (cdahlqvist) [reply README.md]: Corrected.

File: eventdata/challenges/document_id_benchmark.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
-{% set p_bulk_indexing_clients = (bulk_indexing_clients | default(20)) %}
+{% set p_bulk_indexing_clients = (bulk_indexing_clients | default(25)) %}
 {% set p_iterations = bulk_indexing_iterations | default(200000) %}
 {% set p_iterations_per_client = (p_iterations / p_bulk_indexing_clients) | int %}
+{% set p_forcemerge = ( forcemerge | default('no')) | lower %}
```
[the remark this answers is not in the document]
Comment (cdahlqvist) [reply eventdata/challenges/document_id_benchmark.json]: That is indeed much cleaner... Will change.

[the remark this answers is not in the document]
Comment (dliappis) [reply eventdata/challenges/document_id_benchmark.json]: Latest commit looks good!

**Pull request merged**